_posts/2022-10-18-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Trino Summit 2022: Federating humans and data"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector/).
+
+I can't help but bring up the analogy of how Trino federates heterogeneous data
```

Comment:
Thats awfully long for a filler maybe shorten and collapse with topic of next paragraph.

Reply:
Moving this under the Register section.